ephios/event_management/forms.py

```diff
@@ -66,12 +102,13 @@ def save(self, commit=True):
             Group.objects.filter(
                 Q(id__in=self.cleaned_data["visible_for"])
                 | Q(id__in=self.cleaned_data["responsible_groups"])
+                | Q(id__in=self.locked_visible_for_groups)
```

Comment:
this is broken. steps to reproduce:
- have an event that is visible for volunteers and managers and planners are responsible
- visit the edit view with a user in the planner group
- you'll see volunteers in the select field for visible and "also for manager" in the help text
- remove volunteers from the visibility and save the page
- now the event is not visible for managers anymore

Reply:
Very good catch. The locked groups were stored as an *unevaluated* queryset. Evaluation only happened after all permissions had been removed, so the queryset was empty. It is now initialized on form initialization.